### PR TITLE
feat: contact page

### PR DESCRIPTION
**Pull request by MAbdurrehman12**
#### Pre-Flight checklist

- [ ] Screenshots and design review for any changes that affect layout or styling
  - [ ] Desktop screenshots
  - [ ] Mobile width screenshots
- [ ] Testing
  - [ ] Code is tested
  - [ ] Changes have been manually tested

#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/340

#### What's this PR do?
- Responsive Contact page created
- Used bootstrap to keep css minimum
- Contact form is rendered via Wufoo  (some Wufoo limitations are mentioned [here](https://github.com/mitodl/ocw-hugo-themes/issues/340#issuecomment-1015345555))
- [View Deployed PR](https://ocw-hugo-themes-pr-353--ocw-next.netlify.app/contact/)

#### How should this be manually tested?
- Verify that contact nav button in navbar redirects to contact page
- Verify that page is rendered fine on different screen sizes.
- Verify that all links are redirected to appropriate targets.
- Verify that form is working fine and an entry is created in Wufoo each time user submits a form. 

#### Screenshots (if appropriate)
contact page laptop

contact page mobile

### CODE REVIEW

File: base-theme/assets/css/main.scss [hunk 1]
```diff
@@ -74,7 +76,7 @@ h2 {
 
 h3 {
   font-size: 24px;
-  line-height: 2.4rem;
```
Comment (MAbdurrehman12) [inline base-theme/assets/css/main.scss:77]: removing this line-height, to keep the css of h3 generic.
This line-height should be targeted to wherever required

File: www/assets/css/educator.scss [hunk 2]
```diff
@@ -1,11 +1,6 @@
 #educator-main-section {
-  max-width: $max-content-width;
```
Comment (MAbdurrehman12) [inline www/assets/css/educator.scss:2]: Removing this css from here and adding it to .page-single class so it can be reused in multiple pages.

**Comment by pdpinch** [comment]
Is this confirmation page part of your code, or is it provided by wufoo? 

image

**Comment by MAbdurrehman12** [comment]
@pdpinch I got an email of your comment but can't see it here. 
The confirmation part of contact form is also provided by wufoo.

**Comment by pdpinch** [comment]
Sorry, I deleted my comment after I read your note on the issue. 

I’m working on getting access to the wufoo form builder.

File: www/layouts/contact/section.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    </div>
+
+    <div class="row">
+      <div class="col-12">
+        <h3>Get in touch</h3>
+        <p>
+          If you did not find the answer in the <a href="https://ocw.mit.edu/help/">OCW help pages</a>, please fill out the form below
+          and a staff member we will respond as soon as possible.
+        </p>
+      </div>
```
Comment (Wassaf-Shahzad) [inline www/layouts/contact/section.html:81]: Every thing looks good to me. Just one small thing in the invision design the `Get in touch` block and `Still have questions` block probably were merged and the current design does not reflect that.
Also the contact form does not have a heading in the design.


Screenshot 2022-01-21 at 3 48 49 PM

**Review by Wassaf-Shahzad — CHANGES_REQUESTED** [review]

File: www/layouts/contact/section.html (hunk 3 above)
Comment (MAbdurrehman12) [reply www/layouts/contact/section.html:81]: Yeah, just saw it in the design. I'll change it.

Yeah, the contact form does not have a heading but Wufoo form does not allow us to create a form without heading, not sure how it is done in legacy site.

**Review by Wassaf-Shahzad — APPROVED** [review]
LGTM!
Just one thing, The horizontal line is still visible which is not a part of the design However,
I dont think  that can be removed from this from our own css.

**Comment by MAbdurrehman12** [comment]
> LGTM! Just one thing, The horizontal line is still visible which is not a part of the design However, I dont think that can be removed from this from our own css.

Yup, and we will improve the styling later on as discussed [here](https://github.com/mitodl/ocw-hugo-themes/issues/340#issuecomment-1017875895)

**Pull request merged**